gamemodes/terrortown/gamemode/client/cl_keys.lua

```diff
@@ -66,19 +66,40 @@ function GM:PlayerBindPress(ply, bindName, pressed)
             return true
         end
     elseif bindName == "+use" and pressed then
-        if ply:IsSpec() then
-            RunConsoleCommand("ttt_spec_use")
-
-            return true
-        elseif TBHUD:PlayerIsFocused() then
+        if TBHUD:PlayerIsFocused() then
             if ply:KeyDown(IN_WALK) then
                 -- Try to change the access to the button for your current role or team
                 return TBHUD:ToggleFocused(input.IsButtonDown(KEY_LSHIFT))
-            else
-                -- Try to use the button that is currently focused
-                return TBHUD:UseFocused()
```

Comment:
This needs to be reworked as well, as it internally just sends the entity to the server. Not sure if I implement it with this PR or wait for the remote PR to do it together.

Reply:
if you see a benefit to do this in another PR, then it is fine with me

Reply:
I think I leave this PR to focus on just combining all 'use' cases and not changing any behaviour for now.